.config/shell/aliasrc

```diff
@@ -14,9 +14,10 @@ for command in mount umount sv pacman updatedb su shutdown poweroff reboot ; do
 done; unset command
 
 se() {
-	choice="$(find ~/.local/bin -mindepth 1 -printf '%P\n' | fzf)"
-	[ -f "$HOME/.local/bin/$choice" ] && $EDITOR "$HOME/.local/bin/$choice"
-	}
+	s=("${HOME}/.local/bin/"**/*(.))
+	c="$(print -lnr ${s:t:r} | fzf)"
+	[[ "${c}" ]] && "${EDITOR}" ${${(M)s:#*/${c}*}[1]}
```

Comment:
```suggestion
	[[ -z "${c}" ]] && return 1
	[[ "${c}" ]] && "${EDITOR}" ${${(M)s:#*/${c}*}[1]}
```
For purely aesthetic purposes. Presents that error which appears when you press `esc` instead of selecting something.

Very cool. The abbreviated variable names (`s`, `c`) might be confusing for a noob, especially since there's no comment on what the function does. On the other hand, the naming scheme is consistent with a minimalist aesthetic which I dig.

Reply:
Writing the variable directly in brackets have the same meaning on all shells.

`[[ -n "${VARIABLE}" ]]`
`[[ "${VARIABLE}" ]]`

are the same things.

Two brackets are substantially faster but won't work in Dash/sh. Though we already have zsh here.

Reply:
When I first tried this it would send an empty string if I pressed `esc` instead of making a selection. Adding either the `[[ -n ${var} ]]` test or `[[ -z ${var} ]] && return 1` would fix it. After reading your reply I tried the original version again and it seems to work now. This could've been related to some configuration I had back then, although I can't identify which.